src/timedate/timedated.c

```diff
@@ -725,7 +725,7 @@ static int method_set_local_rtc(sd_bus_message *m, void *userdata, sd_bus_error
         if (r < 0)
                 return r;
 
-        if (lrtc == c->local_rtc)
+        if (lrtc == c->local_rtc && fix_system != 0)
```

Comment:
== 0 ? I don't understand this, maybe add a comment.

Reply:
Ouch!

Reply:
Fixed. PTAL.